README.md

```diff
@@ -1,6 +1,6 @@
 [![next](https://github.com/che-incubator/kubernetes-image-puller/actions/workflows/next-build.yml/badge.svg)](https://github.com/che-incubator/kubernetes-image-puller/actions/workflows/next-build.yml)
 
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://github.com/che-incubator/kubernetes-image-puller)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/#https://github.com/che-incubator/kubernetes-image-puller)
```

Comment:
The link works well, but wanted to double check if we want `https://workspaces.openshift.com/` instead of `https://che.openshift.io/`?